cmd/brew-test-bot.rb

```diff
@@ -984,7 +984,7 @@ def test_ci_upload(tap)
     # These variables are for Jenkins and Circle CI respectively.
     pr = ENV["UPSTREAM_PULL_REQUEST"] || ENV["CIRCLE_PR_NUMBER"]
     if pr
-      pull_pr = "#{tap.remote}/pull/#{pr}"
+      pull_pr = "#{tap.remote.sub(/\.git$/, "")}/pull/#{pr}"
```

Comment:
I don't think `test-bot` needs to handle that edge case any more given recent discussions about Linuxbrew/homebrew-science.